### PR TITLE
Replace ncwriter dependency with aodntools

### PR DESCRIPTION
**Pull request by lwgordonimos**

### CODE REVIEW

**Comment by lwgordonimos** [comment]
@mhidas I was playing getting this working in PO box, and thought I may as well put up the PR for you to look over!

File: scripts/install_dependencies.sh [hunk 1]
```diff
@@ -17,6 +17,6 @@ pip install $(aodnfetcher -c ${WHEEL_CACHE_DIR} ${CC_PLUGIN_IMOS_URL} \
     | python -c "import json, sys; print json.load(sys.stdin)['${CC_PLUGIN_IMOS_URL}']['local_file']")
 pip install $(aodnfetcher -c ${WHEEL_CACHE_DIR} ${AODNCORE_URL} \
     | python -c "import json, sys; print json.load(sys.stdin)['${AODNCORE_URL}']['local_file']")
-pip install $(aodnfetcher -c ${WHEEL_CACHE_DIR} ${NCWRITER_URL} \
-    | python -c "import json, sys; print json.load(sys.stdin)['${NCWRITER_URL}']['local_file']")
+pip install $(aodnfetcher -c ${WHEEL_CACHE_DIR} ${AODNTOOLS_URL} \
+    | python -c "import json, sys; print json.load(sys.stdin)['${AODNTOOLS_URL}']['local_file']")
 pip install -r requirements.txt
```
Comment (mhidas) [inline scripts/install_dependencies.sh]: Could add a `pip install httpretty==0.9.6` at the end, just so all the tests pass on a fresh install.

**Comment by mhidas** [comment]
Is this ready to merge?

File: scripts/install_dependencies.sh (hunk 1 above)
Comment (lwgordonimos) [reply scripts/install_dependencies.sh]: Ah yeah, thought I'd fixed that before but didn't. I've added the optional testing requirements in the "standard" way and added to the script.

**Pull request merged**